### ISSUE TITLE
I am getting response code 400.

### ISSUE THREAD

**Issue by Marcoslaurenz** [mdesilva/AmazonFlexUnlimited#172]
I need help I am receiving:

"ERROR: Unable to accept an offer. Request returned status code 400"

I am willing to pay for help

**Comment by jczapatap** [comment]
> I need help I am receiving:
> 
> 
> 
> "ERROR: Unable to accept an offer. Request returned status code 400"
> 
> 
> 
> I am willing to pay for help

Are you using the last update from julio03451? Msg me

**Comment by Marcoslaurenz** [comment]
> > I need help I am receiving:
> > "ERROR: Unable to accept an offer. Request returned status code 400"
> > I am willing to pay for help
> 
> Are you using the last update from julio03451? Msg me

Yes, I am using the latest update of the project, can you help me in any way?

**Comment by Marcoslaurenz** [comment]
I used burpsuite and I got the block acceptance requests to compare it with the project code but I need some help

**Comment by qasimjanjua** [comment]
I am also getting the same error

**Comment by joseph-omidiora** [comment]
Hey @Marcoslaurenz have you been able to fix this error and how did you fix it. If you can help

**Comment by adielp89** [comment]
any update about this ?

**Comment by kpwebdevelopment** [comment]
Has there been any updates on this? Please dm me.

**Comment by ramezray** [comment]
@Marcoslaurenz Did you find a fix to this error?

**Comment by username007-7** [comment]
Has anyone fix this issue with http error code 400 as of today?
@Marcoslaurenz did you get a solution?

**Comment by username007-7** [comment]
> > I need help I am receiving:
> > "ERROR: Unable to accept an offer. Request returned status code 400"
> > I am willing to pay for help
> 
> Are you using the last update from julio03451? Msg me

Where do you find Julios code?
